ui/src/components/PWABadge.vue

```diff
@@ -0,0 +1,118 @@
+<script setup>
```

Comment:
The only thing that would need changing here is to convert this template to an options format instead of a composition api as that is the general format agreed upon in this repository. 

I'd be more than happy to lend a hand if needed!

Reply:
@cstns I have incorporated the converted component into a new file named `PWABadgeOpt.vue`. Could you please review it and verify its accuracy against the original component to confirm that the conversion was executed correctly?

I will replace with original afterwards.

Reply:
Ill try an review it by EOD!